### ISSUE TITLE
Plugin update and compatibility check (July 2024)

### ISSUE THREAD

**Issue by dhawalepranav** [rtCamp/transcoder#288]
We need to complete the following set of tasks for [Blogger to Wordpress](https://wordpress.org/plugins/blogger-to-wordpress-redirection/).

- [x] Make sure it's compatible with the latest WordPress version. (e.g 6.6.X )
- [x] Make sure it's PHP 8.2 compatible.
- [ ] Make sure it's VIP compatible
- [ ] Run the PHPCS scan
- [ ] Review the open GH issues and close them with resolution.
- [ ] Document the user manual with screenshots that will include things like "How to install plugin", "Plugin Pre-requisites" "How to setup plugin for first time", "Plugin Settings with their use case" etc

Before starting the work please add your time estimations for every mentioned task and get it approved by your Project Co-ordinator or Project Manager

**Comment by umeshnevase** [comment]
@dhawalepranav 

- **Make sure it's compatible with the latest WordPress version.** : I'll create a site on rt.gw to test the plugin with the latest WordPress. The plugin needs a hosted site and license to make it work.
- **Make sure it's PHP 8.2 and 8.3 compatible.**:  I've checked the code for PHP PHP 8.2 and 8.3 compatibility by PHPCS. I could not find any issue in the code.
- **Make sure it's VIP compatible**: I've checked VIP compatibility by PHPCS. The code is VIP compatible
- **Run the PHPCS scan**: I've run the PHPCS and I found errors and warnings in the report.
- **Review the open GH issues and close them with resolution.**: I've gone through issues and will start working on it on proper setup. I'll update about estimate and then will start working on it.

**Comment by umeshnevase** [comment]
@dhawalepranav 

**Estimate : 20-24 hours**

I've invested some hours to understand the plugin and setup. Checked code structure. I've created setup on rt.gw server.

- **Make sure it's compatible with the latest WordPress version.** : I've setup plugin with free license on https://transcoder-umesh.rt.gw/ and tested for WordPress 6 compatibility. The plugin is working for both updates. I've to test the plugin with paid version, I'll check with System team for license. `(3-4)`
- **Make sure it's PHP 8.2 and 8.3 compatible.:** I've tested plugin and code on both PHP 8.2 and 8.3, couldn't find any major issue. I found some warning in code. I've listed issue [here] (https://docs.google.com/spreadsheets/d/1bTCxSimhnCjC8AduMI-h8AIZ0-nFaBW8LXZDXl2AE90/edit?gid=0#gid=0) `(3-4)
`
- **Run the PHPCS scan:** I've run the PHPCS and I found errors and warnings in the report. I'll fix the issue and create a PR for that `(2-3h)`
- **Document the user manual with screenshots that will include things like "How to install plugin", "Plugin Pre-requisites" "How to setup plugin for first time", "Plugin Settings with their use case" etc** `(3-4h)`
- **We need to test the transcoder plugin with the rtMedia plugin Latest WordPress and PHP>8.0** `(3-4)`

**Comment by umeshnevase** [comment]
- I've tested plugin with rtMedia, BuddyPress.
- I've tested plugin with latest PHP version and WordPress
- Fixed warning coming on debug.log
- I need to test the transcoder plugin with paid version and paid version of rtMedia.

**Comment by umeshnevase** [comment]
- The plugin is already updated on WordPress plugins repo and it is tested for compatibility with WordPress 6.6.1
- I've tested Transcoder plugin with the license for PHP 8.2 and WordPress 6.6.1. and also tested with rtMedia plugin rtMedia Pro.
- I've fixed PHPCS fixes and warnings.  https://github.com/rtCamp/transcoder/pull/289
- I'm debugging issues pending on repo

**Comment by umeshnevase** [comment]
> Document the user manual with screenshots that will include things like "How to install plugin", "Plugin Pre-requisites" "How to setup plugin for first time", "Plugin Settings with their use case" etc
This is already documented on rtMedia.io site and the document link is provided in the readme file.
https://rtmedia.io/docs/transcoder/

**Comment by umeshnevase** [comment]
I'm working on below issues: 

1. https://github.com/rtCamp/transcoder/issues/287 : This is tested and could not be replicated.
2. https://github.com/rtCamp/transcoder/issues/273 : I'm able to reproduce the issue and I've fixed it will create PR.
3. https://github.com/rtCamp/transcoder/issues/248: I'm getting another issue on my site, it is giving 502 bad gateway. currently Debugging this issue.
